### PR TITLE
fees panel remove input rounding

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes RNBW-####

## What changed (plus any additional context for devs)

remove number rounding on miner tip and max base fee

related to https://linear.app/rainbow/issue/RNBW-2023/input-number-rounding-shouldnt-happen-till-input-loses-focus waiting for confirmation

## Final checklist
[ ] Assigned individual reviewers?
[ ] Added labels?

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]
👍👍👍

File: src/components/expanded-state/custom-gas/FeesPanel.js [hunk 1]
```diff
@@ -232,11 +232,7 @@ export default function FeesPanel({
     let maxBaseFee;
     if (selectedOptionIsCustom) {
       // block more than 2 decimals on gwei value
```
Comment (jinchung) [inline src/components/expanded-state/custom-gas/FeesPanel.js:234]: can we remove this comment as it no longer applies

File: src/components/expanded-state/custom-gas/FeesPanel.js [hunk 2]
```diff
@@ -247,11 +243,7 @@ export default function FeesPanel({
     let maxPriorityFee;
     if (selectedOptionIsCustom) {
       // block more than 2 decimals on gwei value
```
Comment (jinchung) [inline src/components/expanded-state/custom-gas/FeesPanel.js:245]: same can we remove this comment as it no longer applies

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**